api/utils.py

```diff
@@ -199,6 +199,14 @@ def formatEditionOutput(
             edition, records, showAll=showAll, reader=reader
         )
 
+    # Return a list of authors to formatEdition method
+    @staticmethod
+    def workAuthorInfo(edition):
+        workAuthors = []
+        for i in edition.work.authors:
+            workAuthors.append(i['name'])
+        return workAuthors
+
     @classmethod
     def formatEdition(
```

Comment:
This looks really good, the only issue is that this will insert `work_title` and `work_authors` into all edition responses (search, work detail and edition detail), when I think we only want this to appear in the edition detail page.